packages/pf4-component-mapper/src/tests/wizard/wizard.test.js

```diff
@@ -203,6 +203,7 @@ describe('<Wizard />', () => {
   });
 
   it('should call enter handler when pressing enter', () => {
+    // eslint-disable-next-line no-import-assign
```

Comment:
Maybe we should set this rule for all testing eslint files.

Reply:
It was used only in this file. I would like to keep it and use jest spyOn instead